Source/WebCore/Modules/WebGPU/GPU.cpp

```diff
@@ -63,4 +63,9 @@ void GPU::requestAdapter(const std::optional<GPURequestAdapterOptions>& options,
     });
 }
 
+GPUTextureFormat GPU::getPreferredCanvasFormat()
+{
+    return GPUTextureFormat::Rgba8unorm;
```

Comment:
This probably should be plumbed through `PAL`, no?

Also, I believe our preferred format is BGRA8.

Reply:
Yeah, the long term solution is to plumb it through PAL. And I have it as BGRA8 on my local build, I'm not changing it here because I like commits that do one thing only.